Tests/CryptoTests/Authenticated Encryption/AES-GCM-Runner.swift

```diff
@@ -143,96 +143,98 @@ class AESGCMTests: XCTestCase {
+                        var tag: [UInt8] = []
+
+                        do {
+                            let key = try orFail { try SymmetricKey(data: Array(hexString: testVector.key)) }
+                            XCTAssertNotNil(key)
+
+                            let nonceData = try Array(hexString: testVector.iv)
+
+                            let nonce: AES.GCM.Nonce
                             do {
-                                let key = try! SymmetricKey(data: Array(hexString: testVector.key))
-                                XCTAssertNotNil(key)
-
-                                let nonceData = try Array(hexString: testVector.iv)
-                                
-                                let nonce: AES.GCM.Nonce
-                                do {
-                                    nonce = try AES.GCM.Nonce(data: nonceData)
-                                } catch {
-                                    XCTAssert(nonceData.count < 12)
-                                    continue
-                                }
+                                nonce = try AES.GCM.Nonce(data: nonceData)
+                            } catch {
+                                XCTAssertLessThan(nonceData.count, 12)
+                                continue
+                            }
 
-                                if testVector.ct.count > 0 {
-                                    ct = try! Array(hexString: testVector.ct)
-                                }
+                            if testVector.ct.count > 0 {
+                                ct = try orFail { try Array(hexString: testVector.ct) }
+                            }
 
-                                if testVector.msg.count > 0 {
-                                    msg = try! Data(hexString: testVector.msg)
-                                }
+                            if testVector.msg.count > 0 {
+                                msg = try orFail { try Data(hexString: testVector.msg) }
+                            }
 
-                                if testVector.aad.count > 0 {
-                                    aad = try! Data(hexString: testVector.aad)
-                                }
+                            if testVector.aad.count > 0 {
+                                aad = try orFail { try Data(hexString: testVector.aad) }
+                            }
 
-                                if testVector.tag.count > 0 {
-                                    tag = try! Array(hexString: testVector.tag)
-                                }
+                            if testVector.tag.count > 0 {
+                                tag = try orFail { try Array(hexString: testVector.tag) }
+                            }
 
-                                let sb = try! AES.GCM.seal(msg, using: key, nonce: nonce, authenticating: aad)
+                            let sb = try orFail { try AES.GCM.seal(msg, using: key, nonce: nonce, authenticating: aad) }
```

Comment:
I'm a bit nervous about the `orFail`s in this `do` block. I think we should aim to remove them, because they are handled in the `catch` below.

Reply:
Agreed. It was a mistake. Will fix.

Reply:
I also removed a few that were inside `XCTAssert*` functions, since those are already set up to catch errors, I believe.